### DISCUSSION TITLE
How can I implement datetime

### DISCUSSION THREAD

**Discussion by davidsaenz2003** [glideapps/glide-data-grid#560] [Q&A] [answered]
How can I implement datetime, I have a problem with the datetime input for date and time, it does not allow me to save this data, any suggestions? This is the code for the cell-type of datetime:

**Comment by jassmith** [comment, marked as answer]
Can you please check your code? Also the `cells` accessory package does include a date cell.

**Reply by davidsaenz2003** [reply]
Don't worry, I was able to find a solution, I had some errors when implementing datetime, thank you very much
